cmd/verify.go

```diff
@@ -47,18 +48,29 @@ var debugCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		var validateOpts *server.VerifySnpOpts
-		if len(teeNonce) != 0 {
-			validateOpts = &server.VerifySnpOpts{
-				Validation:   server.SevSnpDefaultValidateOpts(teeNonce),
-				Verification: &sv.Options{},
+
+		var validateOpts interface{}
+		switch attestation.GetTeeAttestation().(type) {
+		case *pb.Attestation_TdxAttestation:
+			validateOpts = &server.VerifyTdxOpts{
```

Comment:
I may be missing something about TDX attestation here or go-tdx-guest, but where and how does the nonce get verified?